Kconfig

```diff
@@ -213,6 +213,7 @@ depends on FEATURE_RUNNABLE_PROGRAMS
 source "src/bin/sol-fbp-generator/Kconfig"
 source "src/bin/sol-fbp-runner/Kconfig"
 source "src/bin/sol-fbp-to-dot/Kconfig"
+source "src/bin/sol-tools/Kconfig"
```

Comment:
your call... if you want 2 binaries, then we have `src/bin/sol-gpio` and `src/bin/sol-aio`... or we have a single binary with multiple subcommands.